test/e2e/send-eth-with-private-key-test/send-eth-with-private-key.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable */
```

Comment:
Can we be more specific about what we're disabling? If all we need it to add globals, we can do that too:<sup>[\[1\]][1]</sup>

```suggestion
/* global ethereumjs, Web3 */
```

  [1]:https://eslint.org/docs/user-guide/configuring#specifying-globals